manifests/init.pp

```diff
@@ -305,6 +311,9 @@
   String $jobs_service = $::foreman::params::jobs_service,
   Stdlib::Ensure::Service $jobs_service_ensure = $::foreman::params::jobs_service_ensure,
   Boolean $jobs_service_enable = $::foreman::params::jobs_service_enable,
+  Boolean $jobs_sidekiq = $::foreman::params::jobs_sidekiq,
```

Comment:
Should these all be `dynflow_sidekiq` to keep them congruent with the service and configuration names?

Reply:
I am also a bit worried it may be confusing to users if they are able to set `jobs_service_enable: true` and `jobs_sidekiq: true` but then only the dynflow-sidekiq process is running. Their configuration settings would not reflect the state of the system. We should maybe then have these two options conflict as a validation in `init.pp` ?

Reply:
In current state, we are giving the option(s) new meaning. That is - `job_service_enable` - enable the service, it depends on `jobs_sidekiq` what service it's going to be.

Reply:
If I set `jobs_sidekiq` to true and `jobs_service_enable` to false, I think the current logic still enables the `dynflow-sidekiq` processes which would be unexpected based on `job_service_enable` controlling whether services are enabled or not.

I also still heavily lean towards naming conventions matching closer to the service, e.g. `jobs_sidekiq` becoming `jobs_dynflow_sidekiq`.